dev/NavigationView/NavigationView.cpp

```diff
@@ -220,13 +221,16 @@ NavigationView::NavigationView()
 
     m_navigationViewItemsFactory = winrt::make_self<NavigationViewItemsFactory>();
 
-    s_NavigationViewItemRevokersProperty =
-        InitializeDependencyProperty(
-            L"NavigationViewItemRevokers",
-            winrt::name_of<winrt::IInspectable>(),
-            winrt::name_of<winrt::NavigationViewItem>(),
-            true /* isAttached */,
-            nullptr /* defaultValue */);
+    if (!s_NavigationViewItemRevokersProperty)
+    {
+        s_NavigationViewItemRevokersProperty =
```

Comment:
Pre-existing issue: this is a multi-threaded race condition. Obviously it's pretty narrow now because you have to have two threads both trying to create a NavView at exactly the same time ... but maybe use std::call_once to synchronize the initialization?